File: async-scala-client/src/main/scala/io/swagger/client/model/AdditionalInformation_procRef.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class AdditionalInformation_procRef (
  nProc: Option[String],  // Process Identifier
indProc: Option[String]  // Process Origin - &#39;0&#39; # SEFAZ; - &#39;1&#39; # Justiça Federal; - &#39;2&#39; # Justiça Estadual; - &#39;3&#39; # Secex/RFB; - &#39;9&#39; # Outros 
)

File: javascript-closure-angular-client/API/Client/AdditionalInformationProcRef.js
goog.provide('API.Client.AdditionalInformation_procRef');

/**
 * @record
 */
API.Client.AdditionalInformationProcRef = function() {}

/**
 * Process Identifier
 * @type {!string}
 * @export
 */
API.Client.AdditionalInformationProcRef.prototype.nProc;

/**
 * Process Origin - '0' # SEFAZ; - '1' # Justiça Federal; - '2' # Justiça Estadual; - '3' # Secex/RFB; - '9' # Outros 
 * @type {!string}
 * @export
 */
API.Client.AdditionalInformationProcRef.prototype.indProc;

/** @enum {string} */
API.Client.AdditionalInformationProcRef.IndProcEnum = { 
  0: '0',
  1: '1',
  2: '2',
  3: '3',
  9: '9',
}
